.github/workflows/linting.yml

```
@@ -0,0 +1,26 @@
+name: Lint code
+on:
+  pull_request:
+    branches:
+      - main
+    types: [opened, reopened]
+jobs:
+  run-linters:
+    name: Run linters
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v3
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
```

To match the [build version](https://github.com/Calinator444/newwwie.com/blob/e9aa33a2d98b290fb6d4e9a5af093dd8229d614f/.github/workflows/build.yml#L15)
```suggestion
          node-version: 20.x
```